pkg/virt-controller/watch/application.go

```diff
@@ -603,6 +603,7 @@ func (vca *VirtControllerApp) initCommon() {
 		vca.clusterConfig,
 		vca.launcherSubGid,
 		vca.exporterImage,
+		services.WithEventRecorder(vca.vmiRecorder),
```

Comment:
Do we agree on removing this in a follow up?
In favor of extracting the sidecar registration? (e.g. https://github.com/kubevirt/kubevirt/pull/10423).

Reply:
Fine by me.